apps/android/just.config.js

```diff
@@ -1,3 +1,3 @@
-const { preset } = require('@uifabricshared/build-native');
+const {preset} = require('@fluentui-react-native/scripts');
```

Comment:
super nit: I think there should be spaces around "preset"? not sure why prettier didn't catch it.

Reply:
Yeah, VS Code is removing the spaces I add...

Reply:
But not in another just.config below... maybe there's a config in this specific package?

Reply:
The android test app has a .pretterrc.js. Should I remove it?
```
module.exports = {
  bracketSpacing: false,
  jsxBracketSameLine: true,
  singleQuote: true,
  trailingComma: 'all',
};
```

Reply:
Ah interesting. Yeah I think the prettier should be consistent across the repo, but that can be its own PR